frontend/src/components/campAdmin/CampMaterialListsItem.vue

```diff
@@ -54,6 +54,15 @@ export default {
   props: {
     materialList: { type: Object, required: true },
     disabled: { type: Boolean, default: false }
+  },
+  methods: {
+    deleteErrorHandler (e) {
+      if (e?.response?.status === 422 /* Validation Error */) {
+        return this.$tc('components.camp.campMaterialListsItem.deleteError')
```

Comment:
Why don't we say, what the reason is, instead of just a generic message?

Reply:
I assume it's because it currently doesn't work, see #2764 which he opened around the same time as this PR

Reply:
The main motivation was, because the backend error comes without translation. So I kind of assume that all 422 errors are due to fact that the list was not empty (there's no other deletion validation at the moment).

Would certainly be cleaner to implement proper translation codes.